core/vulkan/vk_virtual_swapchain/cc/virtual_swapchain.cpp

```diff
@@ -294,6 +305,64 @@ void VirtualSwapchain::Destroy(const VkAllocationCallbacks* pAllocator) {
   functions_->vkDestroyCommandPool(device_, command_pool_, pAllocator);
 }
 
+void VirtualSwapchain::DumpImageToFile(uint8_t* image_data, size_t size) {
```

Comment:
This doesn't look like it writes the alpha out. Could we write that to a second file beside the colour data, if it exists?

Reply:
Yeah PPM does not support alpha. In Amber we used lodepng (https://lodev.org/lodepng/) to produce PNG, which support alpha. Could this be a better option? (Lodepng might be problematic if performance is an issue here)

Reply:
Would recommend as a follow up commit, not an extension to this one.